packages/admin/dashboard/src/routes/profile/profile-detail/components/profile-general-section/profile-general-section.tsx

```diff
@@ -62,14 +62,15 @@ export const ProfileGeneralSection = ({ user }: ProfileGeneralSectionProps) => {
             ?.display_name || "-"}
         </Text>
       </div>
-      <div className="grid grid-cols-2 items-center px-6 py-4">
+      {/* TODO: Do we want to implement usage insights in V2? */}
```

Comment:
Yes, probably something to add almost right after the release

Reply:
Alright I'll create a ticket for it in linear 👍